app/src/main/java/com/example/qpc1/newquestion/Question.java

```diff
@@ -0,0 +1,84 @@
+package com.example.qpc1.newquestion;
+
+public class Question {
+    private int id;
+    private String question;
+    private String resultA, resultB, resultC, resultD;
```

Comment:
shoud break to 4 line
private String resultA;
private String resultB;
private String resultC;
private String resultD